misc/scripts/deploy-tools.sh

```diff
@@ -60,6 +62,42 @@ function deployvm {
     done
 }
 
+function deployplugin {
+    for ip in $IP_LIST
+    do
+        TARGET=root@$ip
+        log "Cleaning up older files from $TARGET if exists..."
+        $SSH $TARGET rm -fr $TMP_LOC
+        $SSH $TARGET $MKDIR_P $TMP_LOC
+        $SSH $TARGET $MKDIR_P $BUILD_LOC
+        log "Copying required files to $TARGET ..."
+        $SCP $PLUGIN_BIN $TARGET:$BUILD_LOC
+        $SCP $MANAGED_PLUGIN_SRC $TARGET:$TMP_LOC
+        setupVMType
+        $SSH $TARGET 'cd ' $PLUGIN_LOC '; make info clean plugin'
+        if [ $? -ne 0 ]; then
+            case $FILE_EXT in
+            deb)
+                log "installing make on ubuntu machine"
```

Comment:
I think it's easier to install make (if needed) upfront. Something like 'if [ -z "$(which make)" ] l; then <install> ; fi'
instead of trying, installing and trying again